backend/inspirehep/orcid/builder.py

```diff
@@ -18,6 +19,8 @@
 
 _ELEMENT_MAKERS = {"work": _WORK, "common": _COMMON}
 
+XML_TAG_RE = re.compile(r"<[^>]+>")
```

Comment:
I don't think this is correct, what about a title like `Hubble tension between z < 5 and z > 9`?

Reply:
You are right, I didn't thought about this case. Thanks for the suggestion.